apps/tenant-manager-cli/src/Infrastructure/Operations/TenantOperationsDataProvider.php

```
@@ -0,0 +1,58 @@
+            'code' => $tenant->getCode(),
+            'name' => $tenant->getName(),
+            'status' => $tenant->getStatus(),
+        ];
+    }
+
+    public function exists(string $tenantId): bool
+    {
+        return $this->domainTenantQuery->findById($tenantId) !== null;
+    }
+
+    public function getSettings(string $tenantId): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)